parser/parser/cc/lexer.rl

```diff
@@ -1334,6 +1334,7 @@ void lexer::set_state_expr_value() {
   # Interpolations with immediate variable names simply call into
   # the corresponding machine.
 
+  # TODO(jez) What is this for?
```

Comment:
I assumed this was for string interpolation, though why we would need a completely separate rule for it is not obvious to me.

Reply:
luckily we had a test for this, so i fixed it.